Gems/ROS2/Code/Source/Frame/ROS2Transform.cpp

```diff
@@ -38,7 +53,20 @@ namespace ROS2
         { // Only publish static transforms once
             return;
         }
-        ROS2Interface::Get()->BroadcastTransform(CreateTransformMessage(transform), m_isDynamic);
+        auto transformMessage = CreateTransformMessage(transform);
+        ROS2Interface::Get()->BroadcastTransform(transformMessage, m_isDynamic);
```

Comment:
I'd love to be able to publish tf messages to `robotX/tf` but not to `/tf`

Reply:
Since this is global and has to be known before we create publishers, I think that adding a reg setting is best